src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -2,13 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections;
 using System.Diagnostics;
 using System.Globalization;
 using System.Runtime.CompilerServices;
-using System.Runtime.ConstrainedExecution;
 using System.Runtime.InteropServices;
-
```

Comment:
nit: Add back new line.

I have been separating `using Internal.*` and the rest of the using directives with a new line. Most of the places I have seen do this but it is inconsistent.

Although, I don't think it is worth making the change given CI is mostly green already.